src/PostageStamp.sol

```diff
@@ -222,8 +275,6 @@ contract PostageStamp is AccessControl, Pausable {
             lastUpdatedBlockNumber: block.number
         });
 
-        require(normalisedBalance > 0, "normalisedBalance cannot be zero");
```

Comment:
you removed from here, but you placed this check in `createBatch`. Why?

Reply:
Its here same as with copybatch you just didnt see it :)

![image](https://github.com/ethersphere/storage-incentives/assets/8969767/6eca89a8-8fe8-45e8-8445-824389501476)